stdlib/public/Concurrency/DiscardingTaskGroup.swift

```diff
@@ -431,8 +488,28 @@ public func withThrowingDiscardingTaskGroup<GroupResult>(
 /// be the case with a ``TaskGroup``.
 ///
 /// ### Cancellation behavior
```

Comment:
Same start as the other ones, but then much more discussion because that's the complex one that DOES cancel on throws.